lti_consumer/api.py

```diff
@@ -34,6 +35,10 @@ def _get_or_create_local_lti_config(lti_version, block_location,
     the XBlock to be the source of truth for the LTI version, which is a user-centric perspective we've adopted.
     This allows XBlock users to update the LTI version without needing to update the database.
     """
+    # Replace CCX block locator with master course block locator to allow CCX launch to work.
```

Comment:
```suggestion
    # Replace CCX block locator with master course block locator to allow CCX launch to work.
    # The reason behind the change is to avoid creating a new LtiConfiguration for CCXs and reuse Master's for which its client_id is already recognized by the tool provider.
    
    
```